ExcelWriter.java

```diff
@@ -0,0 +1,69 @@
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.*;
+import javax.swing.JFileChooser;
+
+import org.apache.poi.ss.usermodel.Cell;
```

Comment:
So if you import org.apache.poi.ss.usermodel won't it import all the things under it? I think you can use this throughout.

Reply:
There might be a `*` syntax you can use. Something like

```java
import org.apache.poi.ss.usermodel.*; // <-- note the *
```

Reply:
done